docs/UserGuide.md

```diff
@@ -52,53 +53,60 @@ VolunSync is a **desktop app for keeping track of volunteers and volunteering ev
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
-  e.g `des/ DESCRIPTION`
-
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `reset` and `exit`) will be ignored.<br>
+* Items in square brackets are optional.<br>
+  e.g [des/ DESCRIPTION]
+
+* Commands without parameters (like `help`) ignore any additional input.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+* When copying multi-line commands from this guide, make sure the spaces between lines remain intact, especially if you're using a PDF version.
 </div>
 
+### General Commands
+
 ### Viewing help : `help`
```

Comment:
For the headings `General Commands`, `Event-related commands` and `Volunteer-related commands`, perhaps we could use heading 2 (i.e. `##`) instead of heading 3 (`###`) so that the formatting will look nicer on the table of contents.

Current state:
![image](https://github.com/user-attachments/assets/b282ce43-1dbd-40ca-a7cd-aeed021a18d5)

Edited:
![image](https://github.com/user-attachments/assets/cd178377-0fdd-43d1-abb2-e2eee81458ad)

Reply:
Updated headers